donkeycar/parts/controller.py

```diff
@@ -229,6 +228,108 @@ def set_deadzone(self, val):
         self.dead_zone = val
 
 
+class RCReceiver:
```

Comment:
It looks like you merged parts of the RCReceiver branch into this PR - is this intended or an accident?